akka-docs/src/main/paradox/typed/interaction-patterns.md

```diff
@@ -89,65 +128,87 @@ The function is running in the receiving actor and can safely access state of it
    separate, actor
 
  
-## 1:1 Request-Response with ask between two actors
+## Request-Response with ask between two actors
  
 In an interaction where there is a 1:1 mapping between a request and a response we can use `ask` on the `ActorContext` to interact with another actor.
 
 The interaction has two steps, first we need to construct the outgoing message, to do that we need an @scala[`ActorRef[Response]`]@java[`ActorRef<Response>`] to put as recipient in the outgoing message. The second step is to transform the `Response` or the failure to produce a response, into a message that is part of the protocol of the sending actor.
```

Comment:
"The second step is to transform the `Response` or the failure to produce a response" i don't think we need the last to produce a response